R/glue_apa_results.R

```diff
@@ -193,6 +193,7 @@ est_glue <- function(x) {
 }
 
 stat_glue <- function(x) {
+  if(is(x, "tbl_df")) x <- as.data.frame(x)
```

Comment:
While this works, I think we should commit to the fact that we are working with standard `data.frame`s internally. Therefore, to avoid slightly different behaviors and side effects of `tibble` vs. `data.frame`, I think we should call `as.data.frame()` immediately after calling `broom::tidy()` (or `broom::glance()`). We could even write a wrapper:
```
wrap_tidy <- function(x, ...) {
  as.data.frame(
    broom::tidy(x, ...)
    , stringsAsFactors = FALSE    # for R versions < 4.0
  )
}
```

Reply:
Although I generally agree, this functionality is exposed to users via `glue_apa_results()` and we thus have no control over what what get's passed into this function.

Reply:
Ah, agreed